src/core/data-cite.js

```diff
@@ -68,6 +73,10 @@ function requestLookup(conf) {
         } else {
           wrapInner(elem, anchor);
         }
+        if (!path && !frag) {
+          wrapInner(cite, anchor);
```

Comment:
`wrapInner` is for elements that already have their children. You can just `cite.append(anchor)` in this case.